doc/content/guides/getting-started/console/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Console"
+description: ""
+weight: 8
+---
+
+<style>
+img {
+  border: 1px solid #ddd;
+}
+</style>
+
+## Login
+
+You need to be logged in inside the Console in order to create gateways, applications, devices and API keys. With the stack running in one terminal session, open https://localhost:8885 in your browser.
```

Comment:
I thought we decided `http://localhost:1885` since that is the default config of the stack?